web/src/pages/ShotAnalyzer/components/ShotChart.css

```
@@ -37,50 +39,1731 @@
+
+  .shot-chart-mobile-actions__status > div {
+    min-width: 0;
+    text-align: left;
+  }
+
+  .shot-chart-compare-main-card--mobile {
+    --shot-chart-static-tooltip-compare-preview-height: calc(
+      15.25rem - var(--shot-chart-mobile-compare-height-reduction)
+    );
+    height: calc(
+      var(--shot-chart-mobile-card-height) - 3.5rem - var(--analyzer-mobile-card-gap) -
```

Stylelint (17.13.0)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)